lib/constants.js

```diff
@@ -6,6 +6,7 @@ exports.VERSION = pkg.version;
 
 exports.DEFAULT_PORT = process.env.PORT || 9876;
 exports.DEFAULT_HOSTNAME = process.env.IP || 'localhost';
+exports.DEFAULT_LISTEN_IP = process.env.IP || '0.0.0.0';
```

Comment:
How about we use `KARMA_IP`, I think env variables like that should always be scoped.